src/pep257/__init__.py

```diff
@@ -61,7 +68,7 @@ def next(obj, default=nothing):
 
 
 __version__ = '0.7.1-alpha'
-__all__ = ('check')
+__all__ = ('check',)
```

Comment:
IMO, this should be a separate pull request, fixing https://github.com/GreenSteam/pep257/pull/150